.github/workflows/check-published-links-weekly.yml

```diff
@@ -0,0 +1,43 @@
+name: Check published links weekly
+on:
+  schedule:
+    # 5:17 AM every Sunday
+    - cron: '17 5 * * 0'
+  # Allow for manually running also.
+  workflow_dispatch:
```

Comment:
We would like to add a trigger for PRs to run this on only the files modified in the PR